app/views/payola/transactions/_checkout.html.erb

```diff
@@ -50,7 +51,9 @@
     email: email,
     verify_zip_code: verify_zip_code,
     billing_address: billing_address,
-    shipping_address: shipping_address
+    shipping_address: shipping_address,
+    bitcoin: bitcoin,
+    capture: capture,
```

Comment:
That's true, sorry about that. Fixed now! @peterkeen any idea how to add the capture option to auth payments in a easy way? IS the only thing I'm missing :cry: 

Thanks!